spec/models/vm_reconfigure_request_spec.rb

```diff
@@ -16,7 +16,7 @@
   context '#my_zone' do
     it "with valid source should have the VM's zone, not the requests zone" do
       vm_vmware.update_attributes(:ems_id => ems_vmware.id)
-      request.update_attributes(:options => {:src_ids => [vm_vmware.id]})
```

Comment:
This shouldn't change in this PR, we have always expected ids to be integers, not strings.  I see that the UI currently sends strings, do you know when and why that changed?